src/test/java/teammates/test/cases/testdriver/BackDoorTest.java

```diff
@@ -103,8 +104,26 @@ public void testDeletion() {
         assertEquals(Const.StatusCodes.BACKDOOR_STATUS_SUCCESS, status);
         verifyAbsentInDatastore(courseNoEvals);
 
-        // ----------deleting Feedback Session entities-------------------------
-        // TODO: do proper deletion test
+    }
+
+    @Test
+    public void testFeedbackSessionDeletion() {
```

Comment:
It's odd to have a `testDeletion()` and a `testFeedbackSessionDeletion()` method.